board/freescale/imx8mm_evk/spl.c

```diff
@@ -282,3 +282,34 @@ int do_reset(cmd_tbl_t *cmdtp, int flag, int argc, char * const argv[])
 
 	return 0;
 }
+
+#ifdef CONFIG_SPL_MMC_SUPPORT && SECONDARY_BOOT_RUNTIME_DETECTION
```

Comment:
This is broken currently.  To work correctly this would need to be:
```
#if defined(CONFIG_SPL_MMC_SUPPORT) && defined(CONFIG_SECONDARY_BOOT_RUNTIME_DETECTION)
```

And if we add `CONFIG_SPL_MMC_SUPPORT` to the Kconfig depends we can drop it here:
```
#ifdef CONFIG_SECONDARY_BOOT_RUNTIME_DETECTION
```

Reply:
yeah, right